IntegrationExample/IntegrationExampleApp.swift

```diff
@@ -11,7 +11,11 @@ import SwiftUI
 struct IntegrationExampleApp: App {
     
     @UIApplicationDelegateAdaptor(AppDelegate.self) var appDelegate
-    
+
+    init() {
+        appDelegate.simulateHeadphoneConnection()
```

Comment:
as just discussed offline, let's add a "Headphone Connectivity" section in under the "Processing" view where the connectivity state can be toggled (default is on). the processing session is then activated/deactivated based on the connectivity state. the "MimiProcessingParameters" view is also hidden/shown based on the procession session state.